### PR TITLE
Feature/5 implement logging

### PR DESCRIPTION
**Pull request by Firto**

### CODE REVIEW

File: BobManager.API/BobManager.API/Controllers/ValuesController.cs [hunk 1]
```diff
@@ -3,17 +3,21 @@
...
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
 
 namespace BobManager.API.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
     public class ValuesController : ControllerBase
     {
+
```
Comment (ander21) [inline BobManager.API/BobManager.API/Controllers/ValuesController.cs]: Remove empty line, please!

File: BobManager.API/BobManager.API/Controllers/ValuesController.cs [hunk 2]
```diff
@@ -3,17 +3,21 @@
...
         // GET api/values
         [HttpGet]
         public ActionResult<IEnumerable<string>> Get()
         {
+
+            throw new Exception("lox");
```
Comment (NerevitVasyan) [inline BobManager.API/BobManager.API/Controllers/ValuesController.cs]: remove

File: BobManager.API/BobManager.API/Controllers/ValuesController.cs [hunk 3]
```diff
@@ -3,17 +3,21 @@
...
     [Route("api/[controller]")]
     [ApiController]
     public class ValuesController : ControllerBase
     {
+
         // GET api/values
         [HttpGet]
         public ActionResult<IEnumerable<string>> Get()
         {
+
```
Comment (ander21) [inline BobManager.API/BobManager.API/Controllers/ValuesController.cs]: Remove empty line, please!

File: BobManager.API/BobManager.API/Controllers/ValuesController.cs [hunk 4]
```diff
@@ -3,17 +3,21 @@
...
+            throw new Exception("lox");
             return new string[] { "value1", "value2" };
         }
```
Comment (ander21) [inline BobManager.API/BobManager.API/Controllers/ValuesController.cs:20]: Remove empty line, please!

File: BobManager.API/BobManager.API/Startup.cs [hunk 5]
```diff
@@ -22,9 +24,18 @@ namespace BobManager.API
 {
     public class Startup
     {
+        private readonly FileLogger fileLogger = new FileLogger();
         public Startup(IConfiguration configuration)
         {
             Configuration = configuration;
+
```
Comment (viviviviviivilll) [inline BobManager.API/BobManager.API/Startup.cs:31]: Empty line

File: BobManager.API/BobManager.API/Controllers/ValuesController.cs (hunk 2 above)
Comment (Firto) [reply BobManager.API/BobManager.API/Controllers/ValuesController.cs]: rosolved

File: BobManager.API/BobManager.API/Startup.cs (hunk 5 above)
Comment (Firto) [reply BobManager.API/BobManager.API/Startup.cs:31]: Resolved

**Review by ander21 — COMMENTED** [review]
Resolve please!

**Review by NerevitVasyan — APPROVED** [review]
Approved

**Review by ander21 — APPROVED** [review]
Approve!

**Review by Liosh-a — APPROVED** [review]
Approve

**Pull request merged**